java/core/src/main/java/com/redhat/rhn/common/util/MethodUtil.java

```diff
@@ -210,7 +173,49 @@ public static Object callMethod(Object o, String methodCalled,
         }
     }
 
+    private static boolean searchRightMethod(Class[] types, Object[] converted, Object... params) {
```

Comment:
No need to use var args here, and you should parameterize the `Class`. I would also put `converted` as the last argument, since it's an output parameter:

```suggestion
    private static boolean searchRightMethod(Class<?>[] types, Object[] params, Object[] converted) {
```

Reply:
fixed